app/viewModels/paginationP5/Items.scala

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package viewModels.paginationP5
+
+case class Items(items: Seq[Item], firstItemDotted: Boolean, lastItemDotted: Boolean)
+
+object Items {
+
+  def apply(metaData: MetaData, href: String, additionalParams: Seq[(String, String)]): Items =
+    metaData match {
+      case MetaData(_, _, _, currentPage, totalNumberOfPages) =>
+        val itemList: Seq[Item] =
+          if (totalNumberOfPages < 6) {
```

Comment:
Is it perhaps worth putting the integer limits in this function into constants or maybe even defining them in app config so we can modify them?